rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistSupportImpl.java

```diff
@@ -0,0 +1,209 @@
+package co.rsk.peg.whitelist;
+
+import static co.rsk.peg.whitelist.WhitelistResponseCode.ADDRESS_ALREADY_WHITELISTED;
+import static co.rsk.peg.whitelist.WhitelistResponseCode.ADDRESS_NOT_EXIST;
+import static co.rsk.peg.whitelist.WhitelistResponseCode.DELAY_ALREADY_SET;
+import static co.rsk.peg.whitelist.WhitelistResponseCode.DISABLE_BLOCK_DELAY_INVALID;
+import static co.rsk.peg.whitelist.WhitelistResponseCode.GENERIC_ERROR;
+import static co.rsk.peg.whitelist.WhitelistResponseCode.INVALID_ADDRESS_FORMAT;
+import static co.rsk.peg.whitelist.WhitelistResponseCode.SUCCESS;
+import static co.rsk.peg.whitelist.WhitelistResponseCode.UNKNOWN_ERROR;
+
+import co.rsk.bitcoinj.core.Address;
+import co.rsk.bitcoinj.core.AddressFormatException;
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.bitcoinj.core.Context;
+import co.rsk.bitcoinj.store.BlockStoreException;
+import co.rsk.panic.PanicProcessor;
```

Comment:
Let's remove this, doesn't really do anything but log

Reply:
Done!